### DISCUSSION TITLE
I can't sign in with my twitter account and i don't know why

### DISCUSSION THREAD

**Discussion by SzDyzer** [cssbattle/forum#348] [Bugs reports]
I everybody, i have a little problems for sign in with the account i used usually ( literally all challenge submit ) and is a Twitter account, when i clic in the button of Twitter, he display and disepear after 1 or 2 seconds, i checked in the dev tool in console and i see this error, someone can help me or it's just a bug ? ( sorry for my very bad English btw )

![cssbattle](https://github.com/cssbattle/forum/assets/74179900/564369c1-79b1-4a3a-aea1-920e9a8aff02)

**Comment by chinchang** [comment]
Generally happens when you have third party cookies disabled in the
browser. Check that once and let me know if still doesn’t work. Also don’t
use incognito session.

On Wed, 5 Jul 2023 at 2:52 AM, lamy valentin ***@***.***>
wrote:

> I everybody, i have a little problems for sign in with the account i used
> usually ( literally all challenge submit ) and is a Twitter account, when i
> clic in the button of Twitter, he display and disepear after 1 or 2
> seconds, i checked in the dev tool in console and i see this error, someone
> can be help or it's just a bug ? ( sorry for my very bad English btw )
> [image: cssbattle]
> <https://user-images.githubusercontent.com/74179900/250992616-564369c1-79b1-4a3a-aea1-920e9a8aff02.jpeg>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/cssbattle/forum/discussions/348>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAC4YDVHVQSE4XX3UBUYDOTXOSCSNANCNFSM6AAAAAAZ6E2DJQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
*Kushagra Gour*
kushagra.dev
@chinchang457 <https://twitter.com/chinchang457>

**Reply by SzDyzer** [reply]
Hello, thanks for your answers,
i try everythings i can and nothing works but now i don't know why i tried today and for no reason i can sign in with my account so i hope your solution gonna help someone on the meme case of me
GLHF :)

**Reply by chinchang** [reply]
There was some issue with twitter login for sometime. Its fixed now.
